libnmstate/netapplier.py

```diff
@@ -334,3 +340,26 @@ def _cleanup_iface_ethernet_state_sanitize(desired_state, current_state):
             desired_state.pop('ethernet', None)
             current_state.pop('ethernet', None)
     return desired_state, current_state
+
+
+def _cleanup_iface_ipv6_link_local_addr(desired_state, current_state):
+    for state in (desired_state, current_state):
+        if 'ipv6' not in state:
```

Comment:
This is the opposite of cleanup, more like canonicalization. Another step?
And I think it can be done using:
```
desired_state = _dict_update({'ipv6': {'enabled': False, 'address': []}}, desired_state)
```
Do we need to do this for the current_state?

Reply:
Normally we don't do that for current state, but that require changes to existing unit test cases which does not have ipv6 states.
So I also canonicalize current state.